packages/utils/error-overlay/src/components/Collapsible.js

```diff
@@ -40,15 +40,12 @@ type CollapsiblePropsType = {|
 |};
 
 function Collapsible(props: CollapsiblePropsType): React$Element<'details'> {
-  const [collapsed, setCollapsed] = useState(true);
-
-  const toggleCollapsed = () => {
-    setCollapsed(!collapsed);
-  };
+  const ref = useRef(null);
+  const collapsed = ref.current?.open;
 
   const count = props.children.length;
   return (
-    <details open={!collapsed} onToggle={toggleCollapsed}>
+    <details>
```

Comment:
does this need `ref={ref}`? I don't see where the ref is actually attached to an element.

Reply:
Yes. I was too sloppy converting from a proof of concept